core/src/test/scala/org/apache/spark/rdd/AsyncRDDActionsSuite.scala

```diff
@@ -27,7 +27,7 @@ import org.scalatest.BeforeAndAfterAll
 import org.scalatest.concurrent.Timeouts
 import org.scalatest.time.SpanSugar._
 
-import org.apache.spark.{LocalSparkContext, SparkContext, SparkException, SparkFunSuite}
+import org.apache.spark._
```

Comment:
Sure, that's just your IDE defaults though, doesn't mean it's the right thing in every case. I don't think I'd change imports just to change them, not just because the IDE did it.